### PR TITLE
fix logout, deps cycle, ci  

### PR DESCRIPTION
**Pull request by OfryL**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: .github/workflows/app_test.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ jobs:
 
       - run: npm run linter
 
-      - run: npm run ts
\ No newline at end of file
+      - run: npm run ts || echo "Warning! npm run ts failed."
```
Comment (OmerGery) [inline .github/workflows/app_test.yml:31]: @OmerGery 
Let this be a warning only for now until fully fixed. we can monitor linter this way

File: examples/client/Locomotion/src/pages/Account/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useContext, useState } from 'react';
 import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
 import { PaymentIcon } from 'react-native-payment-icons';
 import { Platform } from 'react-native';
+import Auth from '../../services/auth';
```
Comment (OmerGery) [inline examples/client/Locomotion/src/pages/Account/index.js:5]: @OmerGery

File: examples/client/Locomotion/src/pages/Account/index.js [hunk 3]
```diff
@@ -173,7 +174,7 @@ const AccountContent = () => {
           onSubmit={async () => {
             try {
               await deleteUser();
-              navigationService.navigate(MAIN_ROUTES.LOGOUT);
+              await Auth.logout();
```
Comment (OmerGery) [inline examples/client/Locomotion/src/pages/Account/index.js:177]: @OmerGery

File: examples/client/Locomotion/src/pages/AuthScreens/AuthLoadingScreen.js [hunk 4]
```diff
@@ -4,16 +4,14 @@ import Config from 'react-native-config';
 import moment from 'moment';
 import { Platform } from 'react-native';
 import { APP_ROUTES, MAIN_ROUTES } from '../routes';
-import Auth from '../../services/auth';
+import logout from '../../services/logout';
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/AuthScreens/AuthLoadingScreen.js]: if it's a function it's not a service

Comment (avivbarniv) [reply examples/client/Locomotion/src/pages/AuthScreens/AuthLoadingScreen.js]: Do regular export to the function and export it not as default

**Review by lieblichaaron — APPROVED** [review]

**Review by sephina — APPROVED** [review]

**Pull request merged**